authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+In Ethereum, the following economy-user and node roles (client software types) exist:
+
+|Name       |Role                                                  |Incentivized|Optional|
+|-----------|------------------------------------------------------|------------|--------|
+|Beacon node|Full node of all shards. Passes on data to validators.|No          |No      |
+|Validator  |Proposes and attests to blocks based on beacon data.  |Yes         |No      |
+|VDF ASIC   |Enhances RANDAO to provide better randomness.         |No          |Yes     |
+
+Beacon nodes can potentially be incentivized by being archive nodes of all shards, which means they keep even the data which was wiped due to state rent expiration. A state-rent wiped contract can be restored by an archive node, for a fee. Archive nodes as-a-service are probable in Ethereum 2.0.
+
+In Polkadot:
+
+|Name       |Role                                                        |Incentivized|Optional |
+|-----------|------------------------------------------------------------|------------|---------|
+|Collator   |Full node of particular shard. Passes on data to validators.|Possibly    |No       |
+|Validator  |Full client of all parachains and parathreads. Proposes and attests to blocks based on collator data.      |Yes         |No       |
+|Nominator  |Nominates a validator through which to proxy-stake.         |Yes         |In theory|
+|Fishermen  |Monitor the network for misdeeds like equivocation or illegal randomness.|Yes|Yes|
+
+Validators can be fishermen, which makes fishermen optional. Nominators are _theoretically_ unnecessary, i.e. the network would work fine without them if the validators had enough self-stake to keep themselves in the validator pool.
+
+## Inflation
+
+In both Polkadot and Ethereum the inflation is dynamic and defined as "minimum necessary inflation". The exact number will depend on the amount of staked coins.
+
+## Validators and staking economy
+
+### Polkadot
+
+In Polkadot a validator is a full client (>>FULL OR LIGHT?) of all chains and threads, thus very resource intensive on hard drive space and read/write ops, and on bandwidth.
+
+The assignment of a validator is periodically (>>HOW OFTEN?) reshuffled so that a single validator isn't always responsible for a single parachain. A validator earns DOT from block production (inflation) and transaction fees in a 80/20 spread. 80% of the transaction fee goes to the Polkadot Treasury. The validator gets rewarded twice - they set a fixed number of DOT to get per reward period (epoch - roughly 6 hours), and the rest is then dividied among all the nominators who have nominated this validator, and the validator, based on their respective amounts of staked DOT. It's important to note that nominators can still lose their DOT if they backed an incompetent validator - this is not delegated proof of stake where you simply signal your trust in someone, you literally unlock your DOTs for "burning by malfeasance" when nominating.
```

Comment:
I think we have been using DOTs as the plural of DOT.

Reply:
Just some typos...

Reply:
```suggestion
The assignment of a validator is periodically (>>HOW OFTEN?) reshuffled so that a single validator isn't always responsible for a single parachain. A validator earns DOTs from block production (inflation) and transaction fees in a 80/20 spread. 80% of the transaction fee goes to the Polkadot Treasury. The validator gets rewarded twice - they set a fixed number of DOT to get per reward period (epoch - roughly 6 hours), and the rest is then divided among all the nominators who have nominated this validator, and the validator, based on their respective amounts of staked DOTs. It's important to note that nominators can still lose their DOTs if they backed an incompetent validator - this is not delegated proof of stake where you simply signal your trust in someone, you literally unlock your DOTs for "burning by malfeasance" when nominating.
```